### ISSUE TITLE
Check components and bundles

### ISSUE THREAD

**Issue by mmoreram** [elcodi/elcodi#649]
Check and close Bundles

* Style, CS and Tests
* Naming Conventions
* Distribution of classes and complainant of Standards
* Bundle dependencies
* Functional and Unit tests. Ensure coverage
* READMEs - Blocked by https://github.com/elcodi/elcodi/issues/648
* Composer.json validation. Check dependencies as well.
* Overview documentation

- [ ] Attribute
- [ ] AttributeBundle
- [ ] Banner
- [ ] BannerBundle
- [ ] Cart
- [ ] CartBundle
- [ ] CartCoupon
- [ ] CartCouponBundle
- [ ] Comment
- [ ] CommentBundle
- [ ] Configuration
- [ ] ConfigurationBundle
- [ ] Core
- [ ] CoreBundle
- [ ] Coupon
- [ ] CouponBundle
- [ ] Currency
- [ ] CurrencyBundle
- [ ] Geo
- [ ] GeoBundle
- [ ] Language
- [ ] LanguageBundle
- [ ] Media
- [ ] MediaBundle
- [ ] Menu
- [ ] MenuBundle
- [ ] Metric
- [ ] MetricBundle
- [ ] Newsletter
- [ ] NewsletterBundle
- [ ] Page
- [ ] PageBundle
- [ ] Plugin
- [ ] PluginBundle
- [ ] Product
- [ ] ProductBundle
- [ ] Rule
- [ ] RuleBundle
- [ ] Shipping
- [ ] ShippingBundle
- [ ] Store
- [ ] StoreBundle
- [ ] Tax
- [ ] TaxBundle
- [ ] Template
- [ ] TemplateBundle
- [ ] User
- [ ] UserBundle
- [ ] Zone
- [ ] ZoneBundle

**Comment by xphere** [comment]
Closing in favor of [elcodi/docs](//github.com/elcodi/docs)

**Comment by mmoreram** [comment]
Why are you closing this? :S
